### PR TITLE
Add image to event

### PR DESCRIPTION
**Pull request by luca-rath**

### CODE REVIEW

File: src/Controller/Admin/EventController.php [hunk 1]
```diff
@@ -23,18 +24,25 @@ class EventController extends AbstractRestController implements ClassResourceInt
...
+    private $mediaManager;
+
     /**
      * @var DoctrineListRepresentationFactory
      */
     private $doctrineListRepresentationFactory;
 
     public function __construct(
         EventRepository $repository,
+        MediaManagerInterface $mediaManager,
```
Comment (alexander-schranz) [inline src/Controller/Admin/EventController.php:39]: MediaManager is overloaded and should be avoided for referencing `$mediaRepository` or `$entityManager->getReference` should be used.

Comment (luca-rath) [reply src/Controller/Admin/EventController.php]: fixed

File: src/Controller/Admin/EventController.php [hunk 2]
```diff
@@ -119,12 +127,18 @@ public function deleteAction(int $id): Response
...
     /**
-     * @param string[] $data
+     * @param array<string, mixed> $data
      */
     protected function mapDataToEntity(array $data, Event $entity): void
     {
         $entity->setTitle($data['title']);
 
+        if ($imageId = ($data['image']['id'] ?? null)) {
+            $image = $this->mediaRepository->findMediaById($imageId);
```
[the remark this answers is not in the document]
Comment (luca-rath) [reply src/Controller/Admin/EventController.php:137]: There is already a check for `$imageId`, one line higher

**Comment by luca-rath** [comment]
@nnatter fixed everything

File: src/Controller/Admin/EventController.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (nnatter) [reply src/Controller/Admin/EventController.php:138]: you are right 😇

**Pull request merged**